docs/dev/new_check_howto.md

```diff
@@ -444,23 +444,16 @@ Find below the description for each attributes-each one of them is mandatory-of
 
 ### Building
 
-`setup.py` provides the setuptools setup script that helps us package and build the wheel. To learn more about Python packaging, take a look at [the official python documentation][15]
+`setup.py` provides the setuptools setup script that helps us package and build the wheel. To learn more about Python packaging, take a look at [the official Python documentation][15].
 
 Once your `setup.py` is ready, create a wheel:
-- With the `ddev` tooling (recommended).
-If working on an integration in integrations-extras or integrations-core,
-`ddev release build <INTEGRATION_NAME>`, otherwise `ddev release build /path/to/package`
 
-- Without the `ddev` tooling.
-  ```
-  cd {integration}
-  python setup.py bdist_wheel
-  ```
+- With the `ddev` tooling (recommended): `ddev release build <INTEGRATION_NAME>`
+- Without the `ddev` tooling: `cd <INTEGRATION_DIR> && python setup.py bdist_wheel`
 
 ### Installing
 
-To install your integration, you will need to build your python wheel first. See Building section above.
-Then run the following command:
+To install your Integration, you will need to build your Python wheel first (see the Building section above), then run the following command:
```

Comment:
```suggestion
To install your integration, build your Python wheel first (see the [Building](#building) section above), then run the following command:
```

Reply:
Is `Integration` not one of our fancy capitalised words? :)

Reply:
Nope. Only Agent.